.evergreen/install-dependencies.sh

```diff
@@ -104,5 +104,6 @@ fi
 echo "npm location: $(which npm)"
 echo "npm version: $(npm -v)"
 
-npm install "${NPM_OPTIONS}"
+npm install "${NPM_OPTIONS}" --ignore-scripts
+npm run clean-build
```

Comment:
```suggestion
npm install "${NPM_OPTIONS}" --build-from-source
```

(or if that doesn't apply here, I'd be curious as to why that is 🙂 )

Reply:
I wasn't aware that CLI flag existed 🙂